### PR TITLE
Add CAN connectors

### PR DESCRIPTION
**Pull request by kakooloukia**
- Add CAN connectors
- Add Unittests

Documentation already available in connectors.rst

### CODE REVIEW

File: src/pykiso/lib/connectors/cc_pcan_can.py [hunk 1]
```diff
@@ -0,0 +1,309 @@
+##########################################################################
+# Copyright (c) 2010-2021 Robert Bosch GmbH
```
Comment (sebastianpfischer) [inline src/pykiso/lib/connectors/cc_pcan_can.py:2]: Nop!, wrong license used here

File: src/pykiso/lib/connectors/cc_vector_can.py [hunk 2]
```diff
@@ -0,0 +1,197 @@
+##########################################################################
+# Copyright (c) 2010-2021 Robert Bosch GmbH
```
Comment (sebastianpfischer) [inline src/pykiso/lib/connectors/cc_vector_can.py:2]: Same here

**Review by sebastianpfischer — CHANGES_REQUESTED** [review]

**Comment by sebastianpfischer** [comment]
Also, the doc part is missing.

**Review by sebastianpfischer — APPROVED** [review]

**Comment by sebastianpfischer** [comment]
@kakooloukia, the jenkins is also failing

**Review by Pog3k — APPROVED** [review]

**Pull request merged**